cogs/seasonstats.py

```diff
@@ -9,15 +9,13 @@
 from cogs.donations import ClanConverter
 from cogs.boards import MockPlayer
 from cogs.utils.emoji_lookup import number_emojis
-from cogs.utils.error_handler import error_handler
 from cogs.utils.paginator import SeasonStatsPaginator
 from cogs.utils.formatters import TabularData, readable_time, CLYTable, get_render_type
 from cogs.utils.cache import cache, Strategy
-from cogs.utils.db_objects import DatabasePlayer
-
 
 mock = MockPlayer('Unknown', 'Unknown')
 
+
 class Season(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
```

Comment:
Also need to have this:
```py
    async def cog_before_invoke(self, ctx):
        if hasattr(ctx, 'before_invoke'):
            await ctx.before_invoke(ctx)

    async def cog_after_invoke(self, ctx):
        after_invoke = getattr(ctx, 'after_invoke', None)
        if after_invoke:
            await after_invoke(ctx)
``` 
somewhere in the cog, to make sure all methods to assign stuff to ctx are called properly (from guild config: https://github.com/mathsman5133/donationbot/blob/master/cogs/guildsetup.py#L26)

Reply:
added - will push shortly